docs_src/microservices/device/supported/device-onvif-camera/Walkthrough/custom-build.md

```diff
@@ -47,40 +50,41 @@ Configuring pre-defined devices will allow the service to automatically provisio
+3. Optionally, modify the `Name` and `Description` fields to more easily identify the camera. The `Name` is the camera name used when using ONVIF Device Service Rest APIs. The `Description` is simply a more detailed explanation of the camera.
+4. You can also optionally configure the `CustomMetadata` with custom fields and values to store any extra information you would like.
 
-1. To add more pre-defined devices, copy the above configuration and edit to match your extra devices.
+5. To add more pre-defined devices, copy the above configuration and edit to match your extra devices.
 
 
 ### Configure the Device Service
 1. Open the `cmd/res/configuration.yaml` file using your preferred text editor
 
-1. Make sure `secret name` is set to match `SecretName` in `camera.yaml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.
-
-1. Under `secretName`, set `username` and `password` to your camera credentials. If you have multiple cameras copy the `Writable.InsecureSecrets` section and edit to include the new information.
-
-   ```yaml
-   Writable:
-   LogLevel: INFO
-   InsecureSecrets:
-      credentials001:
-         SecretName: credentials001
-         SecretData:
-         username: <Credentials 1 username>
-         password: <Credentials 1 password>
-         mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"
-      credentials002:
-         SecretName: credentials002
-         SecretData:
-         username: <Credentials 2 username>
-         password: <Credentials 2 password>
-         mode: usernametoken    # assign "digest" | "usernametoken" | "both" | "none"
-   ```
-
-   <p align="left">
-      <i>Sample: Snippet from configuration.yaml</i>
-   </p>
+2. Make sure `secret name` is set to match `SecretName` in `camera.yaml`. In the sample below, it is `"credentials001"`. If you have multiple cameras, make sure the secret names match.
+ 
+3. Under `secretName`, set `username` and `password` to your camera credentials. If you have multiple cameras copy the `Writable.InsecureSecrets` section and edit to include the new information.
+
+    !!! warning
+        Be careful when storing any potentially important information in cleartext on files in your computer. In this case, the credentials for the camera(s) are stored in cleartext in the `configuration.yaml` file on your system.
+        `InsecureSecrets` is for non-production use only.
+
+    !!! example - "Sample: Snippet from configuration.yaml"
+        ```yaml
+        Writable:
+        LogLevel: INFO
+        InsecureSecrets:
+           credentials001:
+              SecretName: credentials001
+              SecretData:
+              username: <Credentials 1 username>
+              password: <Credentials 1 password>
+              mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"
```

Comment:
```suggestion
              SecretData:
                 username: <Credentials 1 username>
                 password: <Credentials 1 password>
                mode: usernametoken   # assign "digest" | "usernametoken" | "both" | "none"
```

Reply:
agree. please make this change. (even if the mistake existed before).